src/FileSystem/AndroidFileSystem.cpp

```diff
@@ -50,6 +52,22 @@ std::filesystem::path AndroidFileSystem::FindPath(const std::filesystem::path& p
 	return path;
 }
 
+bool DefaultFileSystem::IsPathValid(const std::filesystem::path& path)
+{
+	if (path.empty())
+		return false;
+
+	std::ifstream file(path);
+
+	if (std::filesystem::exists(path))
+	{
+		file.close();
+		return true;
+	}
+
+	return false;
```

Comment:
This is not how the android filesystem works. Since android sandboxes the filesystem, you need to call into java code to use the android API.
See the rest of the code.